src/client/pages/Offer/Checkout/InsuranceSummaryDetails.tsx

```diff
@@ -198,18 +205,16 @@ const getHouseholdSizeValue = (householdSize: number, textKeys: TextKeyMap) => {
 const getCoInsuredMaybe = (
```

Comment:
funny that it returns "household size" => but still agree that we should name it like this 😅